tests/cli/publish/test_publish.py

```diff
@@ -8,15 +8,22 @@
 
 from hatch.config.constants import PublishEnvVars
 
-pytestmark = [pytest.mark.usefixtures('devpi'), pytest.mark.usefixtures('local_backend_process')]
```

Comment:
Can we revert the style changes?

Reply:
am i wrong to assume that a `hatch fmt` takes care of these things and it is the authority and it has to be committed?

in case it helps, the only non-style changes in this file are the adjusted input prompts for interactively entered passwords.

Reply:
Yes that will fix it usually but the trailing commas are forcing that new style